api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java

```diff
@@ -76,14 +76,23 @@ public CohortDefinition getNumberOfPatientsWhoInitiatedPreTarvByEndOfPreviousMon
             hivMetadata.getPreArtStartDate().getConceptId()));
 
     cd.addSearch("A1I", map(sqlCohortDefinition, "startDate=${startDate},location=${location}"));
-    cd.addSearch("A1III", map(getPatientsWithTransferFromOtherHF(), "locationList=${location}"));
     cd.addSearch(
         "A1II",
         map(
-            getPatientsWithFirstClinicalConsultationOnTheSameDateAsPreArtStartDate(),
-            "endDate=${endDate},location=${location}"));
+            getNumberOfPatientsTransferredInFromOtherHealthFacilitiesDuringCurrentMonthA1(),
+            "onOrAfter=${startDate},onOrBefore=${endDate},location=${location}"));
```

Comment:
The spec says transferred-in before `startDate`. Mapping both `onOrAfter` and `onOrBefore` will limit the range to within the reporting period.
Change the mapping to `onOrBefore=${startDate},location=${location}`.